.gitignore

```diff
@@ -106,3 +106,6 @@ target/
 
 # IPython
 .ipynb_checkpoints
+
+# .orig
+.py.orig
```

Comment:
Yeah, i was wondering why that is not in there by default. I think because git tracks these and of course ignores them. I want to see if my IDE will then gray these out. Forgot to revert that.